### ISSUE TITLE
use bvlc caffe for transfer learning

### ISSUE THREAD

**Issue by kli-nlpr** [amirgholami/SqueezeNext#5]
```
 Check failed: target_blobs.size() == source_layer.blobs_size() (3 vs. 5) Incompatible number of blobs for layer BatchNorm1
```

**Comment by amirgholami** [comment]
You will need to use IntelCaffe to load the pre-trained models. Intel Caffe (https://github.com/intelcaffe/caffe/) optimizes convolution layers followed by Batch norm layers by fusing them together for faster training, and that is why you cannot read the pre-trained models with the bvlc caffe (since it does not have such optimizations)

**Comment by kli-nlpr** [comment]
@amirgholami  Thanks for your response, Could you provide the pretrained model by using bvlc caffe? I think a lot of people are using bvlc caffe rather than intel caffe. Thanks.

**Comment by vvkuryshev** [comment]
I have same problem. I try to make graph file on Movidius NCS and this process if finished with warning "Check failed: target_blobs.size() == source_layer.blobs_size() (3 vs. 5) Incompatible number of blobs for layer BatchNorm1".
